proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -216,6 +216,15 @@ message NewRunInfo {
     temporal.api.common.v1.DataBlob event_batch = 2;
 }
 
+message SyncWorkflowStateMutationAttributes {
+    temporal.server.api.persistence.v1.VersionedTransition exclusive_start_versioned_transition = 1;
```

Comment:
I am now debating if we should just make this a field of `WorkflowMutableStateMutation` so that the mutation definition is self-contained. We can discuss and change this later.

Even if we do ^, we should still keep the two Attributes (Mutation and Snapshot) definition I think, in case new fields need to be added later.